qtopconf.yaml

```diff
@@ -64,6 +66,18 @@ state_abbreviations:
     s: suspended_of_user
     S: suspended_by_the_queue
     shite: testing_of_user
+  pbs:
```

Comment:
Not 100% sure, but my impression is @fgeorgatos wanted them to be customiseable by the user